modules/imap/functions.php

```diff
@@ -1417,6 +1460,27 @@ function snooze_dropdown($output, $unsnooze = false) {
     return $txt;
 }}
 
+if (!hm_exists('tags_dropdown')) {
+function tags_dropdown($context, $headers) {
+    $folders = $context->get('tag_folders', array());
```

Comment:
These seem to be tag names or labels, why do we call them tag folders? Better call them just labels or tags.

Reply:
Done, i'm now calling them `labels`